### PR TITLE
Fix pay fine ui 2

### PR DESCRIPTION
**Pull request by Millsy66**
Fix Pay Fine UI needs to be reviewed

### CODE REVIEW

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 1]
```diff
@@ -3,98 +3,90 @@
...
 	private UI_STATE state;
 
-	
+
 	public PayFineUI(PayFineControl control) {
-		this.control = control;
-		input = new Scanner(System.in);
-		state = UI_STATE.INITIALISED;
-		control.setUI(this);
+	  this.control = control;
```
Comment (jtulip) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: indents are meant to be 4 spaces

**Review by jtulip — COMMENTED** [review]
indents should be 4 spaces

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 2]
```diff
@@ -3,98 +3,90 @@
...
-				
+
 			case PAYING:
-				double amount = 0;
-				String amtStr = input("Enter amount (<Enter> cancels) : ");
-				if (amtStr.length() == 0) {
-					control.cancel();
-					break;
+			  double amount = 0;
+			  String amtStr = input("Enter amount (<Enter> cancels) : ");
```
Comment (jtulip) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: amtStr could better be amountStr

**Review by jtulip — COMMENTED** [review]
D1 - line 14 - indents should be 4
D2 - line 48 - amtStr could be amountStr

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 3]
```diff
@@ -4,42 +4,42 @@
...
 
 
-	public static enum UI_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+	public static enum UiState{ INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
 
-	private PayFineControl control;
+	private PayFineControl payFineControl;
 	private Scanner input;
-	private UI_STATE state;
+	private UiState state;
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: 1 line break between methods

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 4]
```diff
@@ -4,42 +4,42 @@
...
-	public PayFineUI(PayFineControl control) {
-		this.control = control;
+	public PayFineUI(PayFineControl payFineControl) {
+		this.payFineControl = payFineControl;
 		input = new Scanner(System.in);
-		state = UI_STATE.INITIALISED;
-		control.setUI(this);
+		state = UiState.INITIALISED;
+		payFineControl.setUI(this);
 	}
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: 1 line break between classes

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 5]
```diff
@@ -4,42 +4,42 @@
...
 	
 	
-	public void setState(UI_STATE state) {
+	public void setState(UiState state) {
 		this.state = state;
 	}
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: 1 line break between methods

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 6]
```diff
@@ -4,42 +4,42 @@
 public class PayFineUI {
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java:6]: 1 line break between methods

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 7]
```diff
@@ -4,42 +4,42 @@
...
-	private PayFineControl control;
+	private PayFineControl payFineControl;
 	private Scanner input;
-	private UI_STATE state;
+	private UiState state;
 
 	
-	public PayFineUI(PayFineControl control) {
-		this.control = control;
+	public PayFineUI(PayFineControl payFineControl) {
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: method name needs to be amended to lowerCase. UpperClass reserved for classes

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 8]
```diff
@@ -4,42 +4,42 @@
...
 	
 	
-	public void setState(UI_STATE state) {
+	public void setState(UiState state) {
 		this.state = state;
 	}
 
 
-	public void run() {
+	public void Run() {
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java]: make method name lowerCase

File: ITC205_BaseLine_Code/src/PayFineUI.java [hunk 9]
```diff
@@ -4,42 +4,42 @@
...
 			
 			switch (state) {
 			
 			case READY:
-				String memStr = input("Swipe member card (press <enter> to cancel): ");
-				if (memStr.length() == 0) {
-					control.cancel();
+				String memberStr = input("Swipe member card (press <enter> to cancel): ");
+				if (memberStr.length() == 0) {
+					payFineControl.cancel();
```
Comment (lukexjohnson) [inline ITC205_BaseLine_Code/src/PayFineUI.java:34]: Check if 4 spaces across (1 tab) for all indents

**Review by lukexjohnson — COMMENTED** [review]
- Remove any double spaces between classes
- Fix commented method name from UpperCase to lowerCase
- Amend "break" back to "return" in switch statement

**Comment by bcurby** [comment]
Pull request closed - Author will reopen a new request to update this branch to current version of master

**Pull request closed without merging**